__local/grc20_tokens/onbloc/qux/qux.gno

```diff
@@ -0,0 +1,85 @@
+package qux
+
+import (
+	"strings"
+
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/ownable"
+	"gno.land/p/demo/ufmt"
+	pusers "gno.land/p/demo/users"
```

Comment:
how much do you want username-based transfers?
I'm currently considering to encourage contracts to not care while we expect wallets and gnoweb to be able to perform lookups. This could make all your contracts simpler, only manipulating addresses.

Reply:
If wallets(adena or gnokey) and gnoweb does perform lookups, We really like to handle only `std.Address` which makes contracts simpler(what you just said) and may be cheaper gas.

Reply:
Seems like we can resolve this conversation.
https://github.com/gnolang/gno/pull/3580 has been merged.